### PR TITLE
Prepare 3.6.0 release

### PR DESCRIPTION
**Pull request by Didainius**
* This PR uses released version of go-vcloud-director `v2.15.0`
* Populated CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,80 @@
...
+* `datasource/vcd_network_isolated` add support for `metadata` so this field can be retrieved when reading from isolated networks. ([#804](https://github.com/vmware/terraform-provider-vcd/pull/804))
+* `resource/vcd_network_routed` add support for `metadata` so this field can be set when creating/updating routed networks. ([#804](https://github.com/vmware/terraform-provider-vcd/pull/804))
+* `datasource/vcd_network_routed` add support for `metadata` so this field can be retrieved when reading from routed networks. ([#804](https://github.com/vmware/terraform-provider-vcd/pull/804))
+* `resource/vcd_nsxt_ip_set` and `datasource/vcd_nsxt_ip_set` support VDC Groups by inheriting parent VDC
+or VDC Group from Edge Gateway  ([#809](https://github.com/vmware/terraform-provider-vcd/pull/809))
+* Data source `vcd_storage_profile` supports `metadata` so this field can be populated when reading VDC storage profiles. ([#811](https://github.com/vmware/terraform-provider-vcd/pull/811))
+* resource/vcd_nsxt_app_port_profile and datasource/vcd_nsxt_app_port_profile add support
```
Comment (lvirbalas) [inline CHANGELOG.md:46]: Quotes missing.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,80 @@
...
+* `vcd_independent_disk` allows creating additionally shared disks types by configuring `sharing_type` (VCD 10.3+). Also, add update support. Add new disk type `NVME` and new attributes `encrypted` and `uuid`. Import now allows listing all independent disks in VDC ([#789](https://github.com/vmware/terraform-provider-vcd/pull/789), [#810](https://github.com/vmware/terraform-provider-vcd/pull/810))
+
+## BUG FIXES
+* Fixes Issue #754 where VDC creation with storage profile `enabled=false` wasn't working ([#781](https://github.com/vmware/terraform-provider-vcd/pull/781))
+* Fix Issue #611 when read of `vcd_vapp_vm` and `vcd_vm` resource failed when VM isn't found. Now allows Terraform to recreate resource when it isn't found. ([#783](https://github.com/vmware/terraform-provider-vcd/pull/783))
+* Fix Issue #759 where enable_ip_masquerade handling in vcd_vapp_nat_rules resource wasn't correct ([#784](https://github.com/vmware/terraform-provider-vcd/pull/784))
+* Fix bug in **datasource/vcd_nsxt_app_port_profile** where a lookup of a TENANT scope profile could
```
Comment (lvirbalas) [inline CHANGELOG.md:64]: Use quotes instead of bold?

File: .changes/v3.6.0/818-notes.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
 * Improve docs for `vcd_nsxt_network_dhcp` VDC Group support [GH-818]
-* Improves VDC Group guide documentation [GH-818]
\ No newline at end of file
+* Improves VDC Group guide documentation [GH-818]
```
Comment (lvirbalas) [inline .changes/v3.6.0/818-notes.md:2]: Why all the changes above? The viewer doesn't seem to show a diff :)

File: CHANGELOG.md (hunk 1 above)
Comment (Didainius) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 2 above)
Comment (Didainius) [reply CHANGELOG.md]: Added

File: .changes/v3.6.0/818-notes.md (hunk 3 above)
Comment (Didainius) [reply .changes/v3.6.0/818-notes.md:2]: Missing newlines, broke whole CHANGELOG

**Review by lvirbalas — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,80 @@
 ## 3.6.0 (TBC)
 
-Changes in progress for v3.6.0 are available at [.changes/v3.6.0](https://github.com/vmware/terraform-provider-vcd/tree/main/.changes/v3.6.0) until the release.
+## FEATURES
+* **New Data Source:** `vcd_org_group` allows to fetch an Organization Group to use it with other resources ([#798](https://github.com/vmware/terraform-provider-vcd/pull/798))
+* Add `catalog_version`, `number_of_vapp_templates`, `number_of_media`, `is_shared`, `is_published`, `publish_subscription_type` computed fields to catalog resource and datasource  ([#800](https://github.com/vmware/terraform-provider-vcd/pull/800))
+* Update `vcd_catalog` datasource so now it can take org from provider level or datasource level like modern resources/datasources ([#800](https://github.com/vmware/terraform-provider-vcd/pull/800))
+* **New Resource:** `vcd_nsxt_distributed_firewall` manages Distributed Firewall Rules in VDC Groups
+  ([#810](https://github.com/vmware/terraform-provider-vcd/pull/810))
```
Comment (vbauzysvmware) [inline CHANGELOG.md:8]: is this new line inside sentece is ok, as following lines? in other places it is one line and automatically wrapps

**Review by vbauzysvmware — APPROVED** [review]

**Review by mikeletux — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (Didainius) [reply CHANGELOG.md:8]: Looked at github preview https://github.com/vmware/terraform-provider-vcd/blob/bafe71a5e7bc6ee1c9577806b0b76cc2e079edb4/CHANGELOG.md

I think it looks good:

image

**Pull request merged**